packages/playwright-test/src/util.ts

```diff
@@ -294,3 +294,5 @@ export async function normalizeAndSaveAttachment(outputPath: string, name: strin
     return { name, contentType, body: typeof options.body === 'string' ? Buffer.from(options.body) : options.body };
   }
 }
+
+export const pluginLogger = (plugin: TestPlugin) => debug(`pw:plugin:${plugin.name}`);
```

Comment:
lets use pw-test instead of pw? Since its more test-runner specific.

Reply:
Is there a precedent for this? I frequently just do `pw:*` expecting to get all the relevant logs

Reply:
There is a `export const debugTest = debug('pw:test');` in a few places, so I will scope it from there